docs/lakehouse-sink.md

```diff
@@ -213,7 +205,7 @@ You can create a configuration file (JSON or YAML) to set the properties if you
         "inputs": [
           "test-iceberg-pulsar"
         ],
-        "archive": "connectors/pulsar-io-lakehouse-{{connector:version}}.nar",
+        "archive": "connectors/pulsar-io-lakehouse-2.10.0.7.nar",
```

Comment:
`connectors/pulsar-io-lakehouse-{{connector:version}}-cloud.nar`
We shouldn't use specific version for the doc in master branch, right?

Reply:
yes, I will update it.